README.md

```diff
@@ -27,6 +27,7 @@ Options and their defaults:
 |-----------:|------------:|------------:| :------------ |
 |allow_form_params | true | true | Specifies that input can alternatively be specified as `application/x-www-form-urlencoded` parameters when possible. This won't work for more complex schema validations. |
 |allow_query_params | true | true | Specifies that query string parameters will be taken into consideration when doing validation. |
+|allow_get_body | true | false | Allow GET request body, which merge to request parameter. See (#211) |
```

Comment:
Mind keeping this list alphabetized? (Put `allow_get_body` above `allow_query_params`.)